lib/storage/file.js

```diff
@@ -1019,12 +1019,22 @@ File.prototype.getSignedPolicy = function(options, callback) {
  *     link will expire.
  * @param {string=} options.extensionHeaders - If these headers are used, the
  *     server will check to make sure that the client provides matching values.
+ * @param {string=} options.promptSaveAs - If you provide this value the SaveAs 
+ *     prompt will be displayed when accessing the signed url and the file name 
+ *     will be set to the provided value. If options.responseDisposition is set 
+ *     this option is ignored.
+ * @param {string=} options.responseDisposition - If you provide this value, 
```

Comment:
Oops this is for the parameter above this one. But likewise for all of these params, the style of description is strange. Sorry I'm really just nitpicking at this point. It's really ok, you can leave as-is. :)